components/board/commonsysfs/board.go

```diff
@@ -40,7 +40,7 @@ type Config struct {
 }
 
 // RegisterBoard registers a sysfs based board of the given model.
-func RegisterBoard(modelName string, gpioMappings map[int]GPIOBoardMapping) {
+func RegisterBoard(modelName string, gpioMappings map[int]GPIOBoardMapping, useIoctl bool) {
```

Comment:
It feels like this boolean is stuffing functionality never intended for sysfs, as this is character device. Can we move this somewhere else or refactor it a bit more?

Reply:
Eric and I discussed: the way to resolve this is to rename the sysfs boards to "generic linux" boards, because they rely on non-sysfs linux things. 

If there are too many extra booleans to configure things, we should create a configuration object full of booleans and have just one argument in here. but that can come after we've got at least a second boolean.